core/modules/wsched/ChunkTasksQueue.h

```diff
@@ -97,9 +97,9 @@ class ChunkTasks {
     };
 
 private:
-    int _chunkId;          ///< Chunk Id for all Tasks in this instance.
-    bool _active{false};   ///< True when this is the active chunk.
-    bool _resourceStarved; ///< True when advancement is prevented by lack of memory.
+    int _chunkId;                    ///< Chunk Id for all Tasks in this instance.
+    bool _active{false};            ///< True when this is the active chunk.
+    bool _resourceStarved{false};   ///< True when advancement is prevented by lack of memory.
```

Comment:
No, we don't (have guidelines) that I am aware of.  There is one hard rule so far: do not use bracket-style ctor init for member variables that are references -- this is known to cause crashes under older gccs due to documented compiler bug.

Beyond that, John is a big fan of bracket init.  I think most new qserv code will be written in that style.  Since there are not any hard rules (beyond the one above) I default to letting file-by-file consistency win.  Since this file had paren-init style I opted for that.